### ISSUE TITLE
Accessibility Feedback for Midpoint Review [Deb Resolution, Medical Copays]

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#27420]
# Accessibility Feedback for Midpoint Review [Deb Resolution, Medical Copays]

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh 

[I've left comments in UXPin](https://preview.uxpin.com/18073b171729031ac0af64822d31a7427b012d66#/pages/139873795/comment/sitemap/60f05907b83a21018c0629a6?mode=i) for more details!

## Must
- **[Implementation]** Create unique headings under "What you owe to each facility." I recommend appending the string "Copay balance..." to the `h3` "$300" using a `span` so that both the payment amount and facility are contained within the heading making it unique and more easily navigable.
- Underline or provide chevrons to links under "other information that you need" so that they do not rely on color
- **[Implementation]** Use `aria-label` for any repetitive links on the page e.g. "view your balance details" and "resolve your copay bills"

## Should
- **[Implementation]** Lists should be written as lists using `ul role="list"` (add `role="list"` if bullet styling is removed) and `li`. For example the list of facilities under "Contact information for your VA health care facilities.
- Annotate, comment, or create a table of contents for the headings on the page. A number of elements use `bitter` as their font family, so they may be unintentionally written as headings when they should be `p` elements. For example the "300" and "July 2, 2021" on the Copay details page.

## Consider
- If the note around cancelling copays is relatively important, consider adding a unique heading for it so its more discoverable to screen reader users
- **[Implementation]** Use a description list for the copay detail page. For example `<dl><dt>Balance</dt><dd>$300</dd></dl>`. Make sure to use utility classes NOT a `h` element to give the $300 the bold bitter style.
- Use a one thing per page wizard instead of a dynamic wizard that has no submit button. [Reference this issue ticket for full details on accessibility issues.](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/399)

**Comment by rebeccawalsh** [comment]
## Must
- Per our conversation in the prototype, we'll work with you to make sure the implementation of this piece reads the balance + the location, but may not represent that visually. 
- "Other information you might need" is a global drupal component from the [current page](https://staging.va.gov/health-care/pay-copay-bill/). I'm not sure who makes improvements to global drupal patterns. 
- Will communicate it to engineers

## Should
- Can you be more specific about what your ideal design would be here, or if this is implementation feedback?
- Will communicate it to engineers

## Consider
- Pretty sure this note is disappearing before the release of this tool. It's not the only place this cancellation is mentioned, so this is low priority.
- Still unsure if this feedback carries any visual change or if it's just implementation feedback
- The wizard is already built (as it belongs to the FSR) and we're making more sweeping adjustments to how it works in the future. We will keep this in mind when we redo it.

**Comment by joshkimux** [comment]
@rebeccawalsh I've adjusted the original ticket to note which feedback items are for implementation. I'll be sure to do so in the future. Thank you! 🥳

**Comment by joshkimux** [comment]
@caw310 Dumb question for you! Would the "other information you might need" global drupal component's use of color issue belong as a bug ticket in the design system? Or should it be logged elsewhere? 🙇 

<img width="851" alt="Screen Shot 2021-07-21 at 11 06 32 AM" src="https://user-images.githubusercontent.com/14154792/126512270-dc6c4955-7ae6-4d84-8a30-d62ff28465a2.png">

**Comment by caw310** [comment]
@joshkimux , for the time being, assign this to the Design System. We'll look into this and reassign if necessary. Thanks!

**Comment by joshkimux** [comment]
Thank you! I've opened up a ticket at https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/497

**Comment by LindaLi123** [comment]
hello @joshkimux i believe the change you requested has been done, is there anything else we need to do before we can close the ticket?

**Comment by joshkimux** [comment]
If the changes have been complete, we can close this out! 😄
